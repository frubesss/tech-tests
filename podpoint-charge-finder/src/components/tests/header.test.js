import React from 'react';
import Header from '../Header'
import renderer from 'react-test-renderer'

describe('Header Component', () => {

    it('Snapshot Test', () => {
        const tree = renderer.create(<Header/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

});
