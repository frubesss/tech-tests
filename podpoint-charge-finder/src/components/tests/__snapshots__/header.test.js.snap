// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Snapshot Test 1`] = `
<div
  className="header-nav"
>
  <a>
    FAVOURITES
  </a>
</div>
`;
